packages/integration_test/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -475,21 +475,11 @@
 			baseConfigurationReference = 09505407E99803EF7AA92DE7 /* Pods-RunnerTests.debug.xcconfig */;
 			buildSettings = {
 				BUNDLE_LOADER = "$(TEST_HOST)";
-				CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE;
```

Comment:
Remove unnecessary build setting overrides.  I can do this in a separate PR, if you prefer.